client/my-sites/checkout/composite-checkout/lib/create-wpcom-account-before-transaction.ts

```diff
@@ -10,11 +10,19 @@ export async function createWpcomAccountBeforeTransaction(
 	const isJetpackUserLessCheckout = transactionCart.products.some(
 		( product ) => product.extra.isJetpackCheckout
 	);
+	const isGiftingCheckout = transactionCart.products.some(
+		( product ) => product.extra.isGiftPurchase
+	);
 
+	/*
+	 * Temporarily, we treat Gifting as jetpack-userless-checkout to
```

Comment:
Is this a temporary fix or part of our solution? Just wondering if that comment could be confusing.

Reply:
I added temporary because technically this ks not jetpack signup and I think we have to remove sooner or later

Reply:
"Temporary" is always funny to read in code. 😂 I'm fine with the approach but is there an actual task set up somewhere to do this differently? Because if this works, it's unlikely that there will be much motivation to change it.

Reply:
I will create a follow up to use a new flow that let us send a welcome email. But for now I will remove the "temporary"